File: Snippets/General/Avvale Helpers/Extensions/ExtendDataUnitPk.cs
/// DataUnitPk Extensions
/// Extend DataUnitPk functionalities
/// Extensions, DataUnitPk

// Referenced Assembly : WS\Workspace.AWCommons.Commons;
using Workspace.AWCommons.Commons.Extensions;

// Create a DataUnitPk instance
DataUnitPk pk = new DataUnitPk("CubeId", "EntityId", "ParentId", "ConsId", "ScenarioId", "TimeId");

// Get DataUnitPk id/coordinates
string coordinates = pk.ToKey();

// Get DataUnitPk as script string
string scriptString = pk.ToKeyExt(api);

File: Snippets/General/Avvale Helpers/Extensions/ExtendDataUnitPk.vb
/// DataUnitPk Extensions
/// Extend DataUnitPk functionalities
/// Extensions, DataUnitPk

' Referenced Assembly : WS\Workspace.AWCommons.Commons
Imports Workspace.AWCommons.Commons.Extensions

' Create a DataUnitPk instance
Dim pk As New DataUnitPk("CubeId", "EntityId", "ParentId", "ConsId", "ScenarioId", "TimeId")

' Get DataUnitPk id/coordinates
Dim coordinates As String = pk.ToKey()

' Get DataUnitPk as script string
Dim scriptString As String = pk.ToKeyExt(api)